package.json

```diff
@@ -9,10 +9,16 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/webpacker": "5.2.1",
     "arctic_admin": "^3.2.0",
+    "eslint-config-standard": "^16.0.2",
     "stimulus": "^2.0.0"
   },
   "version": "0.1.0",
```

Comment:
can we add some `scripts` in the `package.json` to be able to run these linters? Like `yarn lint `https://classic.yarnpkg.com/en/docs/cli/run/

Reply:
Excellent idea